docs/medical-api/api-reference/document/post-upload-url.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: "Post Upload URL"
+description: "Returns a presigned URL to use for a medical document upload to our servers."
+api: "POST /medical/v1/document/upload-url"
+---
+
+To use this endpoint, you will need to create a prelimiary version of a [DocumentReference](/medical-api/fhir/resources/documentreference), which we will use as a draft before the file can be uploaded to our servers. In this DocumentReference, we made it a mandatory practice to include the `type` field where you can specify the kind of document ([LOINC](https://www.hl7.org/fhir/valueset-doc-typecodes.html), if possible),`description` field to outline a very brief description of the document, and `context` to provide the relevant dates. Besides these mandatory fields, we ask that you try to provide as much context to the file as possible, including the practitioner (`author` field in the DocumentReference) and narrative (`text`). Note that you **do not** need to enter any references to the patient or your organization, as those will be inferred on our end - anything you will enter in these fields will be overwritten. Once your DocumentReference draft meets the minimum criteria, you can execute the endpoint and receive a presigned URL to use for consecutive file upload. Use that URL with a form-based upload, mapping the URL attributes `url` and `fields` to the corresponding fields of the form. Append the file stream and upload the file. See example below for reference.
```

Comment:
would strongly suggest breaking this up into bullet points or something, long paragraphs in technical docs are painful to read, and it's hard to understand what this does at a glance

suggested format:
- outline what this does at a high level
- talk about the the POST
- talk about the upload

can have 3 sections, each with some bullets for details

Reply:
Agreed 💯 . Also, I'd avoid exposing concept of a "draft" Document Reference. We can just mention that the process will be completed once the file is uploaded.